tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -327,6 +328,13 @@ describe('Eventing tests', function() {
     it('Run Eventing Monitoring tests', async function() {
       await eventingMonitoringTest(bebBackend, isSKR);
     });
+
+    it('check subscription with full qualified event type and exact type matching', async function() {
```

Comment:
Why dont we move this test to `function eventingTestSuite(backend, isSKR, testCompassFlow=false)` so all the relevant tests are in the same block?

Reply:
The reason I put it here is the test case only for Event Mesh. Do you mean I check if it is Event Mesh in `eventingTestSuite` function not at the current place?

Reply:
yes, you can check inside `eventingTestSuite `. Btw we can also check if for JetStream as well.